app/models/forecast_snapshot.rb

```diff
@@ -21,6 +21,23 @@ def retrieve_units(prefix)
     m.units_label_short
   end
 
+  # Get series ID for each series
+  def retrieve_seriesId(name)
```

Comment:
Let's use snake case for this one (i.e., `retrieve_series_id`). Rails prefers snake case. We should also change `dataPortalName`, but that would require a db migration, which is probably beyond the scope of this story.